src/content/components/FeatureView/FeatureView.js

```diff
@@ -121,12 +127,19 @@ export class FeatureView extends React.PureComponent {
   renderBugs(bugs) {
     const bugsByRelease = this.sortByRelease(this.state.bugs);
     return (<React.Fragment>
+
+      <h3>Untriaged bugs</h3>
+      <BugList showResolvedOption={false} bulkEdit={true} tags={true} bugs={bugsByRelease.untriaged} columns={[...displayColumns, "cf_status_nightly", "cf_status_beta"]} />
+
       <h3>Uplift candidates</h3>
       <BugList showResolvedOption={false} bulkEdit={true} tags={true} bugs={bugsByRelease.uplift} columns={[...displayColumns, "cf_status_nightly", "cf_status_beta"]} />
 
       <h3>Required for Current Release (Firefox {currentRelease})</h3>
       <BugList showResolvedOption={false} bulkEdit={true} tags={true} bugs={bugsByRelease.current} columns={displayColumns} />
 
+      <h3>Required for Next Release (Firefox {nextRelease})</h3>
+      <BugList showResolvedOption={false} bulkEdit={true} tags={true} bugs={bugsByRelease.next} columns={displayColumns} />
+
```

Comment:
WIth each priority its owen section, do we need priority column (Lin 25) in feature view?

Reply:
ah good point

Reply:
for backlog it's useful since it can have p5 as well

Reply:
Yeah I think we should just leave it for now while people are getting used to the p1/p2/p3 changes also